src/VscProcess.cpp

```diff
@@ -209,13 +235,29 @@ bool VscProcess::KeyString(KeyString::Request& req, KeyString::Response& res)
   return true;
 }
 
+bool VscProcess::vscSettingsSrv(GetVscSettings::Request  &req, GetVscSettings::Response &res)
+{
+  res.srv_ready = srv_ready;
+  res.serial_number = serial;
+  res.firmware_version = firmware;
+  res.radio_power_level = radio_power_level;
+
+  return true;
+}
+
 void VscProcess::processOneLoop(const ros::TimerEvent&)
 {
   // Send heartbeat message to vehicle in every state
   vsc_send_heartbeat(vscInterface, myEStopState);
 
   // Check for new data from vehicle in every state
   readFromVehicle();
+
+  if (have_firmware && have_radio_power_level && have_firmware && !srv_ready)
```

Comment:
```suggestion
  if (have_serial_ && have_firmware_ && have_radio_power_level_ && !srv_ready_)
```
Duplicate firmware check, and to add `have_serial_` to the checks.

Reply:
Replaced duplicate `have_firmware` with `have_serial`, but kept  the non-underscore formatting to conform with the rest of the repo.